src/index.js

```diff
@@ -5,18 +5,24 @@ import App from './App';
 import { CartProvider } from './Context/Cart-Context';
 import { FilterProvider } from './Context/Filter-Context';
 import { WishlistProvider } from './Context/Wishlist-Context';
+import { ProductProvider } from "./Context/Product-Context"
 import reportWebVitals from './reportWebVitals';
+import { makeServer } from "./server"
```

Comment:
Hey prabhat maybe you can use one single file index.js to import all your context at once.